massa-bootstrap/src/tests/tools.rs

```diff
@@ -269,6 +291,79 @@ pub fn assert_eq_bootstrap_graph(v1: &BootstrapableGraph, v2: &BootstrapableGrap
     }
 }
 
+pub fn get_execution_state() -> BootstrapExecutionState {
```

Comment:
Maybe you can add a little bit of documentation to describe what is in this bootstrap execution state at a glance

Reply:
done

Reply:
my idea was more about copying the values in the doc

Reply:
copy which values into which doc ?

Reply:
the values that are hard coded in this function in this functions doc, to allow for better re usability in other tests

Reply:
done